sr_port/op_indincr.c

```diff
@@ -98,7 +99,10 @@ void	op_indincr(mval *dst, mval *increment, mval *target)
 			rval = EXPR_FAIL;
 			break;
 		}
-		v = put_tref(s);
+		if (NULL != s)
+			v = put_tref(s);
+		else
+			assert(EXPR_FAIL == rval);
 		if (EXPR_FAIL == comp_fini(rval, obj, OC_IRETMVAL, &v, &getdst, target->str.len))
```

Comment:
Agree. Will do that before re-pushing changes though there have been a LOT of changes since the last scan (including V63005) so it may not be obvious.

Reply:
As to your question/issue, it's not clear to me that we would get an issue here again. Look at op_indlvadr.c. It has the same issue with respect to 'v' but when the call to comp_fini is made, the value passed as rval is being checked so invalid expressions don't reference v so no issue raised.

Reply:
I suspect we will not get an issue here because of the &v usage. I think the scanner would give the benefit of the doubt to the code and consider v as an output parameter (that will be initialized inside the comp_fini() call). But we know v is an input parameter in this case and so needs to be initialized before use even if the static scanner does not identify it as an issue.

Reply:
Note the scan will (and has) go into comp_fini() to see how and if the value is used there also. It's fairly thorough for what it does.